jbpm-wb-dashboard/jbpm-wb-dashboard-client/src/test/java/org/jbpm/dashboard/renderer/client/TaskDashboardTest.java

```diff
@@ -291,6 +292,18 @@ public void testTasksByCreationDate() {
     public void testTasksByRunningTime() {
         Displayer displayer = presenter.getTasksByRunningTime();
         DataSet dataSet = displayer.getDataSetHandler().getLastDataSet();
+        DisplayerSettings displayerSettings = displayer.getDisplayerSettings();
+
+        ColumnSettings taskDurationSettings = displayerSettings.getColumnSettings(COLUMN_TASK_DURATION);
+        assertNotNull(taskDurationSettings);
+        assertEquals(taskDurationSettings.getValueExpression(), "value/60000");
```

Comment:
@dgutierr Just one more little thing. The first argument of assertEquals have to be the object expected and the second one the object that you are evaluating.

Reply:
Done.